client/state/themes/selectors/get-canonical-theme.js

```diff
@@ -1,6 +1,6 @@
 import { find } from 'lodash';
 import { getTheme } from 'calypso/state/themes/selectors/get-theme';
-
+import { Theme } from 'calypso/types';
```

Comment:
Since this import is not used in the code but just in the documentation, can we import it inline?